### PR TITLE
OPHJOD-258: Expander

### PR DESCRIPTION
**Pull request by ketsappi**
## Description

Add Expander component

## Later
Later if needed, add controlled state; able to remember and control the expanded/collapsed state programmatically.

## Related JIRA ticket

https://jira.eduuni.fi/browse/OPHJOD-258

### CODE REVIEW

File: lib/components/Expander/Expander.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <Disclosure as="div" className="rounded-[20px] border-[3px] border-[#767676] p-4">
+      {({ open }) => (
+        <>
+          <Disclosure.Button className="flex w-full flex-col">
+            <div className="mb-3 flex w-full flex-row justify-between">
+              <span id={labelId} className="text-[20px] font-bold text-[#333333]">
+                {label}
+              </span>
+              <CaretDownIcon className={`${open ? 'rotate-180' : ''}`} />
```
Comment (sauanto) [inline lib/components/Expander/Expander.tsx:22]:
```suggestion
              <CaretDownIcon className={open ? 'rotate-180' : undefined} />
```

File: lib/components/Expander/Expander.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          <Disclosure.Panel role="region" aria-labelledby={labelId}>
+            {children}
+          </Disclosure.Panel>
+        </>
+      )}
+    </Disclosure>
+  );
+};
+
+const CaretDownIcon = ({ className }: { className: string }) => (
```
Comment (sauanto) [inline lib/components/Expander/Expander.tsx:35]:
```suggestion
const CaretDownIcon = ({ className }: { className?: string }) => (
```

**Review by sauanto — APPROVED** [review]

File: lib/components/Expander/Expander.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    label: 'Label',
+    description: 'Description',
+    children: <div>Children here</div>,
+  },
+};
```
Comment (sauanto) [inline lib/components/Expander/Expander.stories.tsx:21]: The link to the design is missing.

**Review by sauanto — APPROVED** [review]

File: lib/components/Expander/Expander.tsx (hunk 1 above)
Comment (ketsappi) [reply lib/components/Expander/Expander.tsx]: Good find. Changed as suggested.

File: lib/components/Expander/Expander.tsx (hunk 2 above)
Comment (ketsappi) [reply lib/components/Expander/Expander.tsx]: Changed like above.

File: lib/components/Expander/Expander.stories.tsx (hunk 3 above)
Comment (ketsappi) [reply lib/components/Expander/Expander.stories.tsx]: Link to the design is now added.

**Pull request merged**